packages/frontend-2/components/project/page/settings/general/block/Access.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <ProjectPageSettingsBlock
+    background
+    title="Access"
+    :icon="LockClosedIcon"
+    :disabled-message="
+      disabled && disabledMessage ? 'You must be a project owner' : undefined
+    "
+  >
+    <template #introduction>
+      <p>
+        Choose how you want to share your projects with others. Select the option that
+        best suits your needs:
+      </p>
+    </template>
+    <FormRadioGroup
+      v-model="selectedOption"
+      :options="radioOptions"
+      :disabled="disabled"
+      @update:model-value="emitUpdate"
+    />
+  </ProjectPageSettingsBlock>
+</template>
+
+<script setup lang="ts">
+import { LockClosedIcon, LinkIcon, GlobeAltIcon } from '@heroicons/vue/24/outline'
+import { FormRadioGroup } from '@speckle/ui-components'
+import { ProjectVisibility } from '~/lib/common/generated/gql/graphql'
+import { graphql } from '~~/lib/common/generated/gql'
+import type { ProjectPageSettingsGeneralBlockAccess_ProjectFragment } from '~~/lib/common/generated/gql/graphql'
+
+graphql(`
+  fragment ProjectPageSettingsGeneralBlockAccess_Project on Project {
+    id
+    visibility
+  }
+`)
+
+const props = defineProps<{
+  project: ProjectPageSettingsGeneralBlockAccess_ProjectFragment
+  disabled?: boolean
+  disabledMessage?: string
```

Comment:
Also not clear why this prop is there if it isn't really used (as a string, at least)